DEVELOPMENT.md

```diff
@@ -95,3 +96,15 @@ relevant place using the [reST include](https://docutils.sourceforge.io/docs/ref
 Type hints should be used in the code wherever possible. Since the 
 documentation shows the function signatures with the type hints 
 there is no need to include additional type information in the docstrings.
+
+
+## Code Climate
+
+Code Climate is integrated with our GitHub flow. Failing the configured rules will yield a pull request not mergable.
```

Comment:
Where are we configuring the rules? Aren't we just using CC out of the box?

Reply:
We are using defaults, but they're kept in the repo in case we want to change something.

Reply:
Do you reckon we could point people there so they can see what rules we use? The yml file isn't particularly readable, so maybe not :s

Reply:
I think it's fine as is. If you end up doing something about the file, you'll need to read actual CC docs. Plus I'd rather keep only as much docs as necessary. :)